### PR TITLE
Updates to cloud-hosted guide-commons

### PR DESCRIPTION
**Pull request by jakub-pomykala**
Updating the cloud-hosted guide-commons to match the ID review

### CODE REVIEW

File: cloud-hosted/before-start-information.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Instructions for this Cloud Hosted Guide
```
Comment (gkwan-ibm) [inline cloud-hosted/before-start-information.md:1]: No capital for "Cloud Hosted Guide"
-> "cloud-hosted guide"

File: cloud-hosted/before-start-information.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+In this guide you will be using a pre-configured environment that runs in containers on the cloud.
+
+On the left side you have the instructions. You can customise the instructions using the bar at the top. Move between steps using either the arrows or the buttons at the bottom. 
+
+On the right side of the window you have the IDE: This is based of Visual Studio Code, it comes with pre installed tools and also has a built in terminal.
```
Comment (gkwan-ibm) [inline cloud-hosted/before-start-information.md:7]: "IDE:" -> "IDE."

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

File: cloud-hosted/before-start-information.md (hunk 1 above)
Comment (jakub-pomykala) [reply cloud-hosted/before-start-information.md]: Done

File: cloud-hosted/before-start-information.md (hunk 2 above)
Comment (jakub-pomykala) [reply cloud-hosted/before-start-information.md]: Done

**Comment by jakub-pomykala** [comment]
Done

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**